rapid7_insightvm/komand_rapid7_insightvm/util/async_requests.py

```diff
@@ -0,0 +1,49 @@
+import aiohttp
+from typing import Collection
+from komand.exceptions import PluginException
+from .shared_resources import *
+
+
+class AsyncRequests:
+
+    def __init__(self, username, password):
+        self.auth = aiohttp.BasicAuth(login=username, password=password)
+
+    def get_async_session(self) -> aiohttp.ClientSession:
+        """
+        Create and return a new aiohttp ClientSession
+        :return: aiohttp ClientSession
+        """
+        return aiohttp.ClientSession(connector=aiohttp.TCPConnector(ssl=False), auth=self.auth)
```

Comment:
```suggestion
        return aiohttp.ClientSession(connector=aiohttp.TCPConnector(ssl=False), auth=self.auth)
```
Did you mean `verify_ssl=False`? Or `ssl=False`?

Reply:
@mhofert-r7 I know we talked about this - did you ever verify whether or not it just disabled ssl cert verification or if it disables ssl altogether?